package.json

```diff
@@ -2,16 +2,25 @@
   "name": "ethereumjs-tx",
   "version": "1.3.7",
   "description": "An simple module for creating, manipulating and signing ethereum transactions",
-  "main": "es5/index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "coverage": "istanbul cover ./test/index.js",
-    "coveralls": "npm run coverage && coveralls <coverage/lcov.info",
-    "lint": "standard",
-    "test": "npm run test:node && npm run test:browser",
-    "test:browser": "karma start karma.conf.js",
-    "test:node": "tape ./test/index.js",
-    "build:es5": "babel index.js --source-root ./ -d ./es5",
-    "build:docs": "documentation build ./index.js --github -f md > ./docs/index.md && contributor"
+    "build": "ethereumjs-config-build",
+    "prepublishOnly": "npm run test && npm run build",
+    "coverage": "ethereumjs-config-coverage",
+    "coveralls": "ethereumjs-config-coveralls",
+    "docs:build": "typedoc --out docs --mode file --readme none --theme markdown --mdEngine github --excludeNotExported src",
```

Comment:
By the way, what do you think about `--gitRevision master` for the docs? see https://github.com/ethereumjs/ethereumjs-util/pull/181 for context

Reply:
I have mixed feelings.

I think it'd be good to keep the docs pointing to their corresponding commit, but we are regenerating them manually, and they can easily get outdated.

Most people will get to them through the repo, so pointing to master will be right in the majority of the cases. It will only be wrong when navigating a specific version.

I guess the best solution would be for some service to regenerate them on every push/merge. Something like netlify. But that'd mean hosting them somewhere else.